### DISCUSSION TITLE
Question : how do i make another week?

### DISCUSSION THREAD

**Discussion by EikkieShaquille** [KadeArchive/Kade-Engine#460] [Q&A]
How do i make another week? ive watched some tutorials but all of them are for the original and i couldnt find one for kade engine.

**Comment by puyoxyz** [comment]
We just got this cool new guide!!!! That I just merged!!! Because I was gonna merge it earlier but forgot!!! Anyways here it is!!! Brand new guide by prokube yeah!!!

----> [COOL GUIDE](https://kadedev.github.io/Kade-Engine/guides/weeks) <----

**Reply by puyoxyz** [reply]
WAIT WHAT THE LINK DOESNT WORK NOOOOOOOOOOOOO???

**Reply by puyoxyz** [reply]
OK I EDITED IT AND FIXED THE LINK THE WORLD IS SAVED YEAHHHHH

**Reply by Sebyal** [reply]
Epic embed fail

**Reply by puyoxyz** [reply]
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO

**Reply by Sebyal** [reply]
:trollface:

**Reply by EikkieShaquille** [reply]
ok so i did what it said but the week wont play

**Reply by puyoxyz** [reply]
well yeah you also need to make the charts and add the songs

**Reply by EikkieShaquille** [reply]
but i did

**Reply by isakube** [reply]
:trollface:
